Backend/setup.js

```diff
@@ -0,0 +1,93 @@
+/*
+Copyright (C) 2018
+Developed at University of Toronto
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+const rls = require('readline-sync');
+const fs = require('fs');
+
+const common = require('./common.js');
+const db = require('./db.js');
+const logger = require('./logger.js');
+const users = require('./users.js');
+
+/**
+ * add an admin account
+ */
+const setupAdminAccount = function () {
+    logger.info('Velocity server setup');
+
+    const username = rls.question('Please enter your username: ');
+    const fname = rls.question('Please enter your first name: ');
+    const lname = rls.question('Please enter your last name: ');
+    const password = rls.question('Enter your password: ', {
+        hideEchoBack: true,
+        mask: '*'
+    });
+    const password2 = rls.question('Confirm your password: ', {
+        hideEchoBack: true,
+        mask: '*'
+    });
+
+    if (password !== password2) {

```

Comment:
we should probably not end their setup if the passwords don't match, we should have this in a do while loop until the 2 passwords match